manifests/cli.pp

```diff
@@ -60,7 +60,16 @@
     owner   => 'root',
     group   => 'root',
     mode    => '0644',
-    content => template('foreman/hammer_etc.yml.erb'),
+    content => epp(
+      'foreman/hammer_etc.yml.epp',
+      {
+        epp_host            => $foreman_url_real,
```

Comment:
`epp_*` clearly expresses, that these Variables are solely meant for this EPP.  Using Variables like `epp_*` the next Developer is less tempted to just throw in plain Variables from out of the running Puppetcode.  Using Variables with `epp_*` there is a clear Border-Interface between running Puppetcode and the Template.  
Especially, when it comes to rename a Variable in the running Puppetcode, this comes handy, because you do not accidently rename the Templatevariable.

Reply:
epp already protects you against this?
https://puppet.com/docs/puppet/7/lang_template_epp.html#epp_variables

> Unlike Embedded Ruby (ERB) templates, EPP templates cannot directly access variables in the calling class without namespacing.

Reply:
As explained, I see this as a good Practice.  Do you want me to remove it anyways?

Reply:
I think it's redundant and makes thing needlessly verbose. As @alexjfisher said, I consider EPP safe enough (especially if you use type validations inside templates).

Reply:
`epp_` removed